dashboard/app/bisect_test.go

```diff
@@ -188,11 +188,11 @@ For information about bisection process see: https://goo.gl/tpsmEJ#bisection
 
 		c.expectEQ(msg.To, []string{
 			"author@kernel.org",
-			"bugs@syzkaller.com",
 			"default@maintainers.com",
```

Comment:
This also looks like a mailing list. Where does it come from? Shouldn't it be on CC?